frontend/src/app/projects/project-detail/project-details.component.html

```diff
@@ -7,10 +7,22 @@
   <div class="flex max-w-full grow basis-1 flex-col gap-2">
     <div class="flex flex-wrap items-stretch gap-2">
       <app-project-metadata class="flex-1"></app-project-metadata>
-      <app-create-readonly-session class="flex"></app-create-readonly-session>
+      @if ((projectService.project$ | async)?.type !== "training") {
+        <app-create-readonly-session class="flex"></app-create-readonly-session>
+      }
+      @if ((userWrapperService.user$ | async)?.beta_tester) {
+        <app-create-provisioned-session
+          class="w-full"
+        ></app-create-provisioned-session>
+      }
     </div>
```

Comment:
There's a `BetaTestingService` which provides `isBetaTester$`, might make sense to use that here.